### PR TITLE
Add missing zh-Hans translations

### PR DESCRIPTION
**Pull request by binggg**
#### Description of what you did:

Add missing zh-Hans translations

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [x] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [x] Admin
- [ ] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [ ] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

**Comment by binggg** [comment]
Hello @lauriejim 

Sorry for the previous commit that was not checked carefully.

This time I remove  these unnecessary keys.

**Comment by soupette** [comment]
@binggg I suggest you to pull the `develop` branch and make your PR against this branch as some keys had been added/modified.

**Comment by binggg** [comment]
> @binggg I suggest you to pull the `develop` branch and make your PR against this branch as some keys had been added/modified.

OK ~

**Comment by alexandrebodin** [comment]
@binggg Hi, we finally merged the latest release with quite a few changes. If you can update your PR it would be awesome !

**Comment by binggg** [comment]
Hi @alexandrebodin 

I just add some new key against develop branch.

**Comment by alexandrebodin** [comment]
Hi @binggg, can you fix the conflicts ;)

**Comment by binggg** [comment]
> Hi @binggg, can you fix the conflicts ;)

OK

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
     "@testing-library/jest-dom": "^4.0.0",
     "@testing-library/react": "^9.1.1",
     "@testing-library/react-hooks": "^2.0.1",
+    "@types/react-test-renderer": "^16.9.0",
```
Comment (alexandrebodin) [inline package.json]: Can you clear this plz

Comment (binggg) [reply package.json]: OK

**Pull request merged**